test/test_fsdp.py

```diff
@@ -83,7 +86,7 @@ def fsdp_main(rank, world_size, args):
         rank
     )
     model.load_state_dict(torch.load(sd_in_fname))
-    model = FSDP(model)
+    model = FSDP(model, use_orig_params=COMPILE)
```

Comment:
just set use_orig_params to True, and add a comment that compile only supports this setting?